pyscriptjs/examples/simple_clock.html

```diff
@@ -14,13 +14,19 @@
     - paths:
       - ./utils.py
     </py-env>
+    <py-config>
+      - autoclose_loader: false
+    </py-config>
   </head>
 
   <body>
     <div class="font-mono">start time: <label id="outputDiv"></label></div>
     <div id="outputDiv2" class="font-mono"></div>
     <div id="outputDiv3" class="font-mono"></div>
     <py-script output="outputDiv">
+# demonstrates how use the global PyScript pyscript_loader
+# to send operation log messages to it
+pyscript_loader.log("HI")
 import utils
 utils.now()
```

Comment:
Total aside, but at some point we should decide whether to standardize on unindented or indented code.

Reply:
what is the standard in the JS world?